Missing camel-k-kamelet-reify version while building kamelet binding

Installing the following kamelet binding (salesforce to log) throws a dependency build error
```
apiVersion: camel.apache.org/v1alpha1
kind: KameletBinding
metadata:
  name: salesforce-binding
spec:
  source:
    ref:
      kind: Kamelet
      apiVersion: camel.apache.org/v1alpha1
      name: salesforce-source
    properties:
      query: "SELECT Id, Name, Email, Phone FROM Contact"
      topicName: Contact
      clientId: "xxxx"
      clientSecret: "xxx"
      userName: "xxxx"
      password: xxx
  sink:
    uri: "log:info"

```
The error doesn't occur with a timer to log kamelet binding.

The dependency error shows on camel-k-operator log
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.apache.camel.k:camel-k-kamelet-reify:jar is missing. @ line 48, column 17
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.apache.camel.k.integration:camel-k-integration:1.4.0-SNAPSHOT (/tmp/kit-c1qqkkvmrs1mg7j0hij0-508256098/maven/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.apache.camel.k:camel-k-kamelet-reify:jar is missing. @ line 48, column 17
```

--- a/support/camel-k-runtime-bom/pom.xml
+++ b/support/camel-k-runtime-bom/pom.xml
@@ -226,6 +226,11 @@
                 <artifactId>camel-k-loader-groovy</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.k</groupId>
+                <artifactId>camel-k-kamelet-reify</artifactId>
+                <version>${project.version}</version>
+            </dependency>
 
             <dependency>
                 <groupId>org.apache.camel.k</groupId>

